packages/test/test-end-to-end-tests/src/test/gc/gcTombstoneAttachmentBlobs.spec.ts

```diff
@@ -268,6 +268,80 @@ describeNoCompat("GC attachment blob tombstone tests", (getTestObjectProvider) =
             assert.strictEqual(response?.status, 200, `Expecting a 200 response`);
             assert(container2.closed !== true, "Container should not have closed");
         });
+
+        /**
+         * Function that inverses the does not reject condition, i.e., asserts for reject instead. Used in these tests
+         * to demonstrate that because of a bug we are not getting the expected results. Once the bug is fixed, these
+         * asserts should start working as expected.
+         */
+        async function assertDoesNotRejectInverse(block: Promise<any>, message?: string | Error) {
+            return assert.rejects(block, message);
+        };
+
+        /**
+         * This test validates that when blobs are de-duped in different containers, these containers can use these
+         * blobs irrespective of whether the original blob is tombstoned. Basically, after uploading a blob, a container
+         * should be able to use it the same way whether it was de-duped or not.
+         * Note: Currently this fails because the behavior is not as expected and so it's expecting an error.
```

Comment:
Are you going to remember to remove this when it's fixed?

Reply:
Do you mean the comment? Maybe I should just not add it to begin with.